Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs

```diff
@@ -55,26 +55,29 @@ protected virtual void OnEnable()
         {
             instance = (Interactable)target;
             eventList = instance.Events;
-           
+
             profileList = serializedObject.FindProperty("Profiles");
             listSettings = InspectorUIUtility.AdjustListSettings(null, profileList.arraySize);
             showProfiles = EditorPrefs.GetBool(prefKey, showProfiles);
 
             SetupEventOptions();
             SetupThemeOptions();
 
-            actionOptions = Interactable.GetInputActions();
-
             enabled = true;
         }
-        
+
         protected virtual void RenderBaseInspector()
         {
             base.OnInspectorGUI();
         }
 
-        public override void OnInspectorGUI()
+        public sealed override void OnInspectorGUI()
```

Comment:
why add sealed here? why would we want no further sub-classes to be able to override?

Reply:
Yep, good question, we have the RenderCustomInspector available for overriding, so that path can be used. I needed to move actionOptions to this location and wanted to prevent overriding of this behavior by a sub class. My thinking is that if there is a compelling reason to undo it that RenderCustomInspector doesn't satisfy, it can be undone then. Today no class overrides it.

Reply:
@keveleigh, @wiwei, thoughts?

Reply:
I think it's fine here, given that the intent is to make sure that the code below:

 if (actionOptions == null && !Interactable.TryGetInputActions(out actionOptions))
            {
                EditorGUILayout.HelpBox("Mixed Reality Toolkit is missing, configure it by invoking the 'Mixed Reality Toolkit > Configure...' menu", MessageType.Error);
            }

Actually gets run. Being able to further override makes it easy to forget to add this (I assume that this check is actually critical).

It might be worth having a minor comment about that (i.e. "please override through RenderCustomInspector to ensure that correct prerequisite state checks are performed on your behalf")

Reply:
Good idea on the comment, will add it.